src/tuner/nccl_ofi_model.c

```diff
@@ -137,7 +137,7 @@ static double all_reduce_ring_cost(struct nccl_ofi_tuner_model_dims const* dims,
 	const double intranode_cost = intranode_latency + (intranode_bytes / intranode_bw);
 
 	const double internode_latency = net_latency * num_steps_internode;
-	const double internode_bw = (NCCL_OFI_TUNER_INTERNODE_BW) * per_proto_bandwidth_overhead;
```

Comment:
the old actually matches our style, but whatever.

Reply:
There's nothing intentional about this ordering -- if you'd prefer it reversed we can do that.

Reply:
Going to leave this as-is for now.